code/modules/projectiles/gun.dm

```diff
@@ -83,6 +83,8 @@
 	var/displays_maptext = FALSE
 	var/can_ammo_display = TRUE
 	var/obj/item/ammo_display
+	var/empty_sound = /decl/sound_category/out_of_ammo
+	var/casing_drop_sound = /decl/sound_category/casing_drop_sound
```

Comment:
Redundant var is redundant - why not just use the casing drop sound...that's already on the casing?

Reply:
Oversight

Reply:
Actually, no. This was no oversight. This is needed so shotgun shells can have their own drop sounds.